kuma/wiki/jinja2/wiki/react_document.html

```diff
@@ -102,6 +102,7 @@
 
   <!-- site js -->
   {% javascript 'react-main' %}
+  {% javascript 'mathml' %}
```

Comment:
This is good and a nice tradeoff. This way, a crash won't block the rest of the JS (bundle) file. 

I still think it should be done with React instead of fragile DOM manipulations but if we just leave this as is, and that frees you up to work on what's more important, then I'm all for greenlighting the review of this. Thank you!